### ISSUE TITLE
simulation environment

### ISSUE THREAD

**Issue by minxuanjun** [adrelino/interpolation-methods#3]
Thank you for your great work! @adrelino 
I want to reproduce your experiment in your thesis work "Continuous time trajectory estimation for 3D SLAM from an actuated 2D laser scan". Can you share your simulation environment?

**Comment by adrelino** [comment]
no, it is proprietary. But nowadays you might be able to re-create something similar in gazebo, BlenderProc or omniverse.
